include/fmt/format.h

```diff
@@ -495,6 +495,16 @@ FMT_CONSTEXPR20 inline auto countl_zero(uint32_t n) -> int {
   return lz;
 }
 
+FMT_CONSTEXPR20 inline auto countl_zero(uint64_t n) -> int {
+#ifdef FMT_BUILTIN_CLZLL
+  if (!is_constant_evaluated()) return FMT_BUILTIN_CLZLL(n);
+#endif
+  int lz = 0;
+  constexpr uint64_t msb_mask = 1ull << (num_bits<uint64_t>() - 1);
+  for (; (n & msb_mask) == 0; n <<= 1) lz++;
+  return lz;
```

Comment:
I would suggest moving this logic into a separate function (e.g. `countl_zero_fallback`) and reusing here and in 32-bit overload.

Reply:
You mean having a template `countl_zero_fallback` that is called from both of the overloads?

Reply:
Yes

Reply:
Anyway, it can be done separately so I'll merge this as is.